packages/components/statistic/src/index.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div :class="[ns.b()]">
+    <div class="head">
+      <slot name="title">
+        <span class="title">
```

Comment:
Class should have prefix, like `:class="ns.e('title')"`.

Reply:
Thank you, I will use css in js style to adjust